components/RichText/RichText.module.scss

```diff
@@ -82,3 +82,30 @@
         color: var(--color--dark-indigo);
     }
 }
+
+.unorderedList {
+    list-style: none;
+    margin-left: 2px;
+}
+
+.listItem {
```

Comment:
**Nickpick**: 
```suggestion
.unorderedListItem {
```

Reply:
I've realised this raises an issue if we want to add ordered lists in the future, I have refactored the code to resolve this & removed the generic list item rich text modifier from `RichText.tsx`